cmake/SetupFlags.cmake

```diff
@@ -72,6 +72,7 @@ INTERFACE
     $<${with_fmath}:
       SINGULARITY_USE_FMATH
     >
+    PORTABILITY_STRATEGY_NONE
```

Comment:
This seems necessary... but `PORTABILITY_STRATEGY_KOKKOS` is *always* provided to `make` as a flag for any set of options I tried.

Reply:
I don't think this should be necessary. very weird.

Reply:
Yeah I think this is unnecessary given the default behavior of `ports-of-call`, I'll remove it